(DOCSP-31686): Updated the Release Notes examples to our standards. (#149)
mongodb/docs-meta: source/style-guide/release-notes-guidelines.txt

Release Notes Guidelines
========================

.. warning::
   

>    This page provides model guidance and examples for Release Notes. 

Author: I want to convey that you should use the content on this page as an example to imitate, but each team does Release Notes in a slightly different way. I tweaked the wording a bit.

>    Verify the correct and consistent model for each product with your 

Reviewer: [suggestion]

Maybe "layout" instead of "model" here?

     - Use sentences to describe the new feature or enhancement.
       Provide as needed, and provide a link to any section in the
       documentation that describes that feature.
     - Supports IPv6. When you create a server, you can specify 
       both IPv4 and IPv6 addresses.

   * - Resolved issues
     - Provide an initial phrase that describes the issue that was
       fixed. Start the phrase with a past-tense verb. If necessary,
       include sentences to further explain the fix. If you list only
       phrases, do not use ending punctuation.
     - - Supports Debian 8 (Jessie)

       - Trimms the Monitoring ID and Monitoring Token configuration
         variables to ensure correctness.

>        - Fixes the Xen Server 6 package repo.

Reviewer 2: Issue:
The guideline doesn't seem to make sense and both the existing and updated examples don't seem to match that guideline.

> Provide an initial phrase that describes the issue that was fixed. Start the phrase with a past-tense verb

Suggestion:

I think this might need to be reworked to what you intended. Personally, I think "Resolved issues" should contain both what the behavior is now and what it used to be. E.g. 
"The driver now allows SRV record addresses that contain a trailing dot. Previous versions treated this as an invalid address."

Applying this to the first example, this could be:
"Supports Debian 8 (Jessie)" => "Now supports Debian 8 (Jessie). Prior versions were incompatible due to ..."

Author: @ccho-mongodb, good catch, I'll change the guidance to "present-tense." "Now" seems redundant in a release notes context. For Atlas, we usually start with a present tense verb.

Reviewer 3: For the Debian example, I don't think it is necessary to do more explaining about why . For the 'Fixes the server' example, it would be nice to go into what was fixed to demonstrate that slightly more explanation is necessary.

   * - Known issues
     - Use sentences to describe the issue. If a workaround is
       available, explain it.
     - Results from the ``$indexStats`` operator exclude 
       queries that use the ``$match`` or ``mapReduce`` functions.

   * - Documentation changes
     - Provide an initial phrase that describes the issue that was
       fixed. Start the phrase with a past-tense verb. If necessary,
       include sentences to further explain the fix. If you list only
       phrases, do not use ending punctuation. Provide a link to the
       relevant section in the documentation.

>      - Clarifies the alert policy descriptions to include how

Reviewer 2: Issue:
The existing and revised example don't match the guidelines.

Suggestions:

1) Fix the guidance to match the examples.

2) Personally, I would interpret "Documentation changes" as "what in the documentation changed?" which would  match with the existing past tense guidance. However, I'd be fine with present tense if it works better for everyone.

Author: @ccho-mongodb, I think of it more like "this release..."

       fixed. Start the phrase with a verb. :ref:`Use present tense 
       <use-present-tense>` or past tense consistently throughout the 
       release notes. If necessary, include sentences to further 
       explain the fix and the previous behavior. If you list only 
       phrases, do not use ending punctuation.
     - - Supports Debian 8 (Jessie).

       - Trims the Monitoring ID and Monitoring Token configuration
         variables to ensure correctness.

       - Fixes the Xen Server 6 package repository to resolve an issue 
         that caused inconsistencies and errors with package 
         management.

   * - Known issues
     - Use sentences to describe the issue. If a workaround is
       available, explain it.
     - Results from the ``$indexStats`` operator exclude 
       queries that use the ``$match`` or ``mapReduce`` functions.

   * - Documentation changes
     - Provide an initial phrase that describes the issue that was
       fixed. Start the phrase with a verb. :ref:`Use present tense 
       <use-present-tense>` or past tense consistently throughout the 
       release notes. If necessary, include sentences to further 

>        explain the fix. If you list only phrases, do not use ending 

Reviewer 2: Suggestion:

I think a documentation change isn't necessarily a fix. Also, I think rewording to use positive sentences could shorten the sentence.

```suggestion
       explain the changes. If you list only phrases, omit ending 
```

       punctuation. Provide a link to the relevant section in the 

>        documentation.

Reviewer 2: Question: Could the example feature a link or placeholder for one since the recommendation is to provide a link? Or should this be optional?

"Clarifies... the alarm state. To learn more about ..., see \<link\>".
